harvard-lts/etd_dash_service
# Etd 190: add healthchecks to container

**add healthchecks to container.**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-190


# What does this Pull Request do?
add healthchecks to container for docker

# How should this be tested?

1) start locally
2) enter container
3) `python scripts/healthcheck.py`  
    you should see a _healthy: last updated X secs ago_ message
4) `echo $?` should return 0.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties: @awoods
